docs/self-managed/platform-deployment/helm-kubernetes/guides/ingress-setup.md

```diff
@@ -98,10 +100,10 @@ helm install demo camunda/camunda-platform -f values-combined-ingress.yaml
 
 Once deployed, you can access the Camunda 8 components on:
 
-- **Web applications:** `https://camunda.example.com/[identity|operate|optimize|tasklist|modeler|console]`
+- **Web applications:** `https://camunda.example.com/[identity|operate|optimize|tasklist|modeler|console|zeebe]`
   - _Note_: Web Modeler also exposes a WebSocket endpoint on `https://camunda.example.com/modeler-ws`. This is only used by the application itself and not supposed to be accessed by users directly.
 - **Keycloak authentication:** `https://camunda.example.com/auth`
-- **Zeebe Gateway:** `grpc://zeebe.camunda.example.com`
+- **Zeebe Gateway:** `https://zeebe.camunda.example.com`
```

Comment:
:thought_balloon: I preemptively applied the changes from Jesse's PR by removing the `grpc://` references.